Documents/start_contribution.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to JDI Light
+
+## Step 1: Clone the Repository 
+
+First, you'll need to clone the repository to your local machine. You can do this with the following command: git clone https://github.com/jdi-testing/jdi-light.git
+
+## Step 2: Navigate to the Project Directory 
+
+Next, navigate to the project directory: cd jdi-light
+
+## Step 3: Understand the Properties Files 
```

Comment:
If think it is better to start from folders description and structure - core, libraries for frameworks, tests for them

Reply:
added breif description